sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -78,8 +58,10 @@ export { Delivery }
 // @public
 export type EntityStatus = "Active" | "Creating" | "Deleting" | "ReceiveDisabled" | "SendDisabled" | "Disabled" | "Renaming" | "Restoring" | "Unknown";
 
+// Warning: (ae-forgotten-export) The symbol "WaitTimeOptions" needs to be exported by the entry point index.d.ts
+//
 // @public
-export interface GetMessageIteratorOptions extends OperationOptions {
+export interface GetMessageIteratorOptions extends OperationOptions, WaitTimeOptions {
```

Comment:
One learning I had from Event Hubs era was the explosion of interfaces that results in discomfort when browsing our docs.

Right now, we are using the rule "if it appears more than once, make new type to re-use"
But the benefits of re-usability does not outweigh the noise of a 100 interfaces.

I would suggest that we not introduce `WaitTimeOptions` and inline the max wait time in both `GetMessageIteratorOptions` and `ReceiveBatchOptions`

Reply:
So interestingly enough these document nicer than before:

https://docs.microsoft.com/en-us/javascript/api/@azure/event-hubs/geteventhubpropertiesoptions?view=azure-node-latest

Note that it's squashing all the interfaces into a single set of fields (ie, showing inherited). So I think most of my concern with that initially isn't really there anymore.

In this case it's less "if it appears more than once re-use" and more "make sure that I'm documenting that this field is literally the same name in all options so I can pass those options to sub-functions". I'd be worried about fat-fingering or missing updating an interface and having the wait time property misalign.